python/ql/src/analysis/DefinitionTracking.qll

```diff
@@ -490,3 +490,24 @@ class NiceLocationExpr extends @py_expr {
         )
     }
 }
+
+/**
+ * Gets an element, of kind `kind`, that element `e` uses, if any.
```

Comment:
```suggestion
 * Gets the definition (of kind `kind`) for the expression `use`, if one can be found.
```

Reply:
This was me attempting to imitate the docstring from the C++ `[definitionOf](https://github.com/github/codeql/blob/master/cpp/ql/src/definitions.qll#L125-L134)`. If python only has one `kind`, then indeed I'm not sure why the original definitions query includes it in the `select`. @tausbn do you know if this is some legacy vestige of something or other?

Reply:
This almost certainly falls into "hysterical raisins" territory. I don't know that anything depends on this kind being present, but I also don't know that it _doesn't_, so I would recommend leaving it in for now.